packages/conf-secp256k1/conf-secp256k1.1.0.0/opam

```diff
@@ -7,9 +7,9 @@ license: "MIT"
 dev-repo: "git+https://github.com/dakk/conf-secp256k1"
 
 depexts: [
-  ["libsecp256k1-0" "libsecp256k1-dev"] {os-family = "debian"}
+  ["libsecp256k1-dev"] {os-family = "debian"}
```

Comment:
Please if these are changed, can we make the canges into a `conf-secp256k1.2` package (no need for the other version numbers, conf- packages are changing very rarely)?

Reply:
Sure, done